com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -46,8 +47,8 @@ public class ControllerVisualizer : MonoBehaviour
         /// <summary>
         /// Cached reference to hands aggregator for efficient per-frame use.
         /// </summary>
-        protected HandsAggregatorSubsystem HandsAggregator => handsAggregator ??= HandsUtils.GetSubsystem();
-        private HandsAggregatorSubsystem handsAggregator;
+        [Obsolete("Deprecated, please use XRSubsystemHelpers.HandsAggregator instead.")]
```

Comment:
Best not to overuse the Obsolete attribute, especially when these are really internal calls.  I see no reason why any consumer would be calling for the HandsAgegator on a visualizer.
Plus this is still a preview, so changes only need to be in the release notes.  Best not to overburden the maintenance of classes and leave code to not be cleaned up later.

Reply:
We've been trying to play it as safe as possible throughout these previews, allowing devs a couple of releases to update their code before forcing a build break on them. Yes, it's slightly more maintenance for us, but we're willing to take that on for ease of consumption!